classes/lang/KeysReference/FeatureFlagLang.php

```diff
@@ -38,3 +38,7 @@
 // Product multi store feature flag in 8.0
 trans('New product page - Multistore', 'Admin.Advparameters.Feature');
 trans('Access the new product page, even in a multistore context. This is a work in progress and some features are not available.', 'Admin.Advparameters.Help');
+
+// Legacy link feature flags
+trans('Attribute group', 'Admin.Advparameters.Feature');
+trans('Enable / Disable migrated attribute group page. This is a work in progress and some features are not available.', 'Admin.Advparameters.Help');
```

Comment:
```suggestion
trans('Enable / Disable migrated attribute group page.', 'Admin.Advparameters.Help');
```
You removed the last part from the fixtures but not here